src/goto-programs/remove_virtual_functions.cpp

```diff
@@ -479,6 +506,9 @@ exprt remove_virtual_functionst::get_method(
   return symbol->symbol_expr();
 }
 
+/// Remove all virtual function calls in a goto program and replace
+/// them with calls to their most derived implementations
+/// Returns true if at least one function has been replaced
```

Comment:
use `\return`

Reply:
If I add `\return` this no longer is considered as `\brief` section, and linter is going to kill me.

Also I really, really don't like the pattern where a function that just returns something has to have its behaviour specified in the `\return` clause only. I don't believe any commercial library does that (except Java, where everything is void-returning).

Reply:
Everything before the first tag is considered as `\brief` (the high-level function description, it can be as long as you like). Yes, I'm missing full stop, thanks for pointing that out 👍